.github/workflows/generate-user-stories.yml

```diff
@@ -71,22 +69,20 @@ jobs:
         with:
           format: list
           query:  label:"v1.0" label:"User Story"
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.USER_TOKEN }}
```

Comment:
Volgens mij hebben we dit juist een keer gewijzigd in 'GITHUB_TOKEN' zoals het nu in de master staat.

Reply:
klopt. Dat is nu gefixt met een merge vanuit main